src/modern/components/participant-info-modal.tsx

```
@@ -18,30 +15,15 @@ type ParticipantInfoModalProps = {
       <p className="text-[11px] font-semibold uppercase tracking-wider text-muted-foreground">{label}</p>
       <div className="flex items-center gap-2">
         <code className="flex-1 overflow-x-auto rounded bg-muted px-2 py-1 font-mono text-xs">{value}</code>
-        <button
-          className="shrink-0 rounded p-1 text-muted-foreground hover:bg-muted hover:text-foreground"
-          onClick={handleCopy}
-          title="Copy to clipboard"
-          type="button">
-          {copied ? <Check className="h-3.5 w-3.5 text-emerald-500" /> : <Copy className="h-3.5 w-3.5" />}
-        </button>
+        <CopyButton
+          className="h-7 w-7 rounded p-1 text-muted-foreground hover:bg-muted hover:text-foreground"
```

ESLint

[error] Expected indentation of 12 space characters but found 10.

(react/jsx-indent-props)